typing/typedtree.mli

```diff
@@ -325,6 +325,12 @@ and expression_desc =
                          (Labelled "y", Some (Texp_constant Const_int 3))
                         ])
 
+            The sort here is _not_ the sort of this application.  It is the sort
+            of a _full_ application of the same function, and is used to
```

Comment:
I don’t know what “full” means here. I think it means an application that includes all omitted arguments up to this point. But if you have an application that’s partial but omits no arguments (because the skipped arguments are after all the ones that are supplied), then the sort is the sort of the application. (This is basically implied below, but it still makes me worried about “full”.)